src/appmixer/system/core/OnAnyFlowComponentError/OnAnyFlowComponentError.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+module.exports = {
+
+    async start(context) {
+
+        // Get current WEBHOOK_FLOW_COMPONENT_ERROR from system config
+        const config = await context.callAppmixer({
+            endPoint: '/config',
+            method: 'GET'
+        });
+        const urls = config.find(c => c.key === 'WEBHOOK_FLOW_COMPONENT_ERROR')?.value?.split(',') || [];
+        // Use enqueueOnly to automatically enqueue the webhook in the `receive` method.
+        const newUrl = context.getWebhookUrl({ enqueueOnly: true });
+        const updatedUrls = [...urls, newUrl].join(',');
+
+        await context.log({ step: 'Update config', newUrls: updatedUrls, oldUrls: urls });
+        // Update WEBHOOK_FLOW_COMPONENT_ERROR in system config
+        await context.callAppmixer({
+            endPoint: '/config',
+            method: 'POST',
+            body: {
+                key: 'WEBHOOK_FLOW_COMPONENT_ERROR',
+                value: updatedUrls
+            }
+        });
+    },
+
+    async stop(context) {
+
+        // Get current WEBHOOK_FLOW_COMPONENT_ERROR from system config
+        const config = await context.callAppmixer({
+            endPoint: '/config',
+            method: 'GET'
+        });
+        const urls = config.find(c => c.key === 'WEBHOOK_FLOW_COMPONENT_ERROR')?.value?.split(',');
+        const newUrl = context.getWebhookUrl();
```

Comment:
Does this work? You don't have the `{ enqueueOnly }` here, so the URL will be different than the one in the `start` and the the filter will not find it, because it does string comparison. So my guess is that the URL will not be unregistered, when the flow is stopped and will stay there forever.

Reply:
Fixed